packages/aws-cdk/lib/commands/flags/operations.ts

```diff
@@ -0,0 +1,480 @@
+  }
+
+  /** Displays detailed information for specific flags matching the given names */
+  private async displaySpecificFlags(flagNames: string[]): Promise<void> {
+    const matchingFlags = this.flags.filter(f =>
+      flagNames.some(searchTerm => f.name.toLowerCase().includes(searchTerm.toLowerCase())));
+
+    if (matchingFlags.length === 0) {
+      await this.ioHelper.defaults.error(`Flag matching "${flagNames.join(', ')}" not found.`);
+      return;
+    }
+
+    if (matchingFlags.length === 1) {
+      const flag = matchingFlags[0];
+      await this.ioHelper.defaults.info(`Flag name: ${flag.name}`);
+      await this.ioHelper.defaults.info(`Description: ${flag.explanation}`);
+      await this.ioHelper.defaults.info(`Recommended value: ${flag.recommendedValue}`);
+      await this.ioHelper.defaults.info(`User value: ${flag.userValue}`);
+      return;
+    }
+
+    await this.ioHelper.defaults.info(`Found ${matchingFlags.length} flags matching "${flagNames.join(', ')}":`);
+    await this.displayFlagTable(matchingFlags);
+  }
+
+  /** Returns sort order for flags */
+  private getFlagSortOrder(flag: FeatureFlag): number {
+    if (flag.userValue === undefined) return 3;
+    if (this.isUserValueEqualToRecommended(flag)) return 1;
+    return 2;
+  }
+
+  /** Displays flags in a formatted table grouped by module and sorted */
+  async displayFlagTable(flags: FeatureFlag[]): Promise<void> {
+    const sortedFlags = [...flags].sort((a, b) => {
+      const orderA = this.getFlagSortOrder(a);
+      const orderB = this.getFlagSortOrder(b);
+
+      if (orderA !== orderB) return orderA - orderB;
+      if (a.module !== b.module) return a.module.localeCompare(b.module);
+      return a.name.localeCompare(b.name);
+    });
+
+    const rows: string[][] = [['Feature Flag Name', 'Recommended Value', 'User Value']];
+    let currentModule = '';
+
+    sortedFlags.forEach((flag) => {
+      if (flag.module !== currentModule) {
+        rows.push([chalk.bold(`Module: ${flag.module}`), '', '']);
+        currentModule = flag.module;
+      }
+      rows.push([
+        `  ${flag.name}`,
+        String(flag.recommendedValue),
+        flag.userValue === undefined ? '<unset>' : String(flag.userValue),
+      ]);
+    });
+
+    const formattedTable = formatTable(rows, undefined, true);
+    await this.ioHelper.defaults.info(formattedTable);
+  }
+
+  /** Checks if a flag has a boolean recommended value */
+  isBooleanFlag(flag: FeatureFlag): boolean {
+    const recommended = flag.recommendedValue;
+    return typeof recommended === 'boolean' ||
+      recommended === 'true' ||
+      recommended === 'false';
+  }
+
+  /** Checks if the user's current value matches the recommended value */
+  private isUserValueEqualToRecommended(flag: FeatureFlag): boolean {
+    return String(flag.userValue) === String(flag.recommendedValue);
+  }
+
+  /** Shows helpful usage examples and available command options */
+  async displayHelpMessage(): Promise<void> {
```

Comment:
Wait, is this still used? You should get this for free when doing `cdk flags --help`.